File: app/src/main/java/com/constraint/vagabond/data/remote/GetRecAreasData.java
package com.constraint.vagabond.data.remote;

import com.constraint.vagabond.data.RecreationalAreaList;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Header;
import retrofit2.http.Query;

public interface GetRecAreasData {

  @GET("recareas")
  Call<RecreationalAreaList> getRecreationalAreaData(
      @Query("query") String query, @Query("full") boolean full, @Header("apikey") String apiKey);
}

File: app/src/main/java/com/constraint/vagabond/main/search/GetRecAreasData.kt
package com.constraint.vagabond.main.search

import com.constraint.vagabond.data.entities.RecreationalAreaList

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Header
import retrofit2.http.Query

interface GetRecAreasData {

    @GET("recareas")
    fun getRecreationalAreaData(
            @Query("query") query: String, @Query("full") full: Boolean, @Header("apikey") apiKey: String): Call<RecreationalAreaList>
}
